app/components/Modal/HtlcModal.jsx

```diff
@@ -1140,42 +997,21 @@ class HtlcModal extends React.Component {
                                 </Form.Item>
                                 <div className="content-block transfer-input">
                                     <div className="no-margin no-padding">
-                                        {/*  F E E  */}
-                                        <div
-                                            id="txFeeSelector"
-                                            className="small-12"
-                                        >
-                                            <AmountSelector
-                                                label="transfer.fee"
-                                                disabled={true}
-                                                amount={fee}
-                                                onChange={this.onFeeChanged.bind(
-                                                    this
-                                                )}
-                                                asset={
-                                                    fee_asset_types.length &&
-                                                    feeAmount
-                                                        ? feeAmount.asset_id
-                                                        : fee_asset_types.length ===
-                                                          1
-                                                            ? fee_asset_types[0]
-                                                            : fee_asset_id
-                                                                ? fee_asset_id
-                                                                : fee_asset_types[0]
-                                                }
-                                                assets={fee_asset_types}
-                                                display_balance={balance_fee}
-                                                // tabIndex={tabIndex++}
-                                                error={
-                                                    this.state
-                                                        .hasPoolBalance ===
-                                                    false
-                                                        ? "transfer.errors.insufficient"
-                                                        : null
+                                        <FeeAssetSelector
+                                            label="transfer.fee"
```

Comment:
label="transfer.fee" is the default, remove across the whole PR is this value is used

Reply:
done